Source/animation/types/animation_generators/Cache/AnimationCache.swift

```diff
@@ -8,7 +8,7 @@ import AppKit
 
 class AnimationUtils {
 
-    class func layerForNodeRenderer(_ renderer: NodeRenderer, _ context: AnimationContext, animation: Animation, customBounds: Rect? = .none, shouldRenderContent: Bool = true) -> ShapeLayer {
+    class func layerForNodeRenderer(_ renderer: NodeRenderer, animation: Animation, customBounds: Rect? = .none, shouldRenderContent: Bool = true) -> ShapeLayer {
```

Comment:
Maybe we should try to use maximum line length equal to 120 symbols (for example)? Long lines are really uncomfortable to read and understand, especially here on Github.

Reply:
Would be great to add it to our [SwiftLint](https://github.com/exyte/Macaw/blob/master/.swiftlint.yml) 😉

Reply:
If you agree - it takes about 5 mines to update rules for SwiftLint and maybe some time to format current issues.